keps/sig-node/20210706-pod-addmission-plugin.md

```diff
@@ -0,0 +1,314 @@
+---
```

Comment:
Please convert your KEP to the [latest template](https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template). I see that the version skew strategy and production readiness questionnaires are missing, among other key sections of the design.

Reply:
Thanks @ehashman for checking. I will update it

Reply:
@ehashman updated this kep as per latest template.